cime/src/drivers/mct/cime_config/namelist_definition_drv.xml

```diff
@@ -4724,4 +4724,29 @@
     </values>
   </entry>
 
+  <entry id="seq_flux_mct_albdif">
+    <type>real</type>
+    <category>seq_flux_mct</category>
+    <group>seq_flux_mct_inparm</group>
+    <desc>
+       Surface albedo for direct radiation
+    </desc>
+    <values>
+      <value>0.06</value>
+      <value aqua_planet_sst_type="sst_aquap11">0.07</value>
+    </values>
+  </entry>
+
+  <entry id="seq_flux_mct_albdir">
+    <type>real</type>
+    <category>seq_flux_mct</category>
+    <group>seq_flux_mct_inparm</group>
+    <desc>
+       Surface albedo for diffuse radiation
+    </desc>
+    <values>
+      <value>0.07</value>
+    </values>
+  </entry>
```

Comment:
This is kinda odd. Comparing with my PR, I added an ATM namelist variable for the constant albedo. What's the argument for adding it to the drv_in file?

Reply:
Probably because the albedo is calculated in the coupler/driver.